clar2wasm/src/tools.rs

```diff
@@ -476,48 +476,54 @@ fn crosseval(snippet: &str, env: TestEnvironment) -> Result<CrossEvalResult, Kno
     }
 }
 
-pub fn crosscheck(snippet: &str, expected: Result<Option<Value>, Error>) {
-    let eval = match crosseval(
-        snippet,
-        TestEnvironment::new(TestConfig::latest_epoch(), TestConfig::clarity_version()),
-    ) {
+fn execute_crosscheck(
+    env: TestEnvironment,
+    snippet: &str,
+    pre_compare: impl FnOnce(&CrossEvalResult),
```

Comment:
Am i missing something or is this never called in this PR? I don't understand the reason this is here.

Reply:
The pre_compare specifically

Reply:
Its called twice. In `crosscheck_compare_only` and `crosscheck_compare_only_with_expected_error`